apps/fulfillment/readme.md

```diff
@@ -0,0 +1,16 @@
+# Fulfillment app
+
+## Mock server
+
+1. Run `nx run fulfillment:init-env:mock` to init env variables for mock server.
+   Variables description:
+   - `ORYX_FULFILLMENT_MOCK_PROXY_URL` - the URL where proxy routes will be redirected
+   - `ORYX_FULFILLMENT_MOCK_PORT` - the port on which the mock server will run (mock server runs on `3000` port if this variable is not defined)
+2. Run `nx run fulfillment:serve` to start mock server.
```

Comment:
Is it a mock server or FA?)

Reply:
Should be `nx run fulfillment:mock` I think 😁

Reply:
Oh, missed it, thanks for mentioning :)